stdlib/public/stubs/GlobalObjects.cpp

```diff
@@ -106,12 +106,36 @@ swift::_SwiftEmptySetStorage swift::_swiftEmptySetStorage = {
   0 // int entries; (zero'd bits)
 };
 
+#if defined(__Fuchsia__)
+#include <zircon/syscalls.h>
+static __swift_uint64_t randomUInt64() {
```

Comment:
This should be using `getentropy` which is provided by Fuchsia's libc, not `zx_cprng_draw` which is a syscall and considered to be implementation detail.

Reply:
Yeah, I wrote this before that was implemented and had our security guys give me an ok. Our getentropy function ends up doing nearly the same thing (except this has some protection if actual read was less than requested unlike our version). I was going to wait for TO-460 was finished and then we can just use std::random_device like every other arch.

Reply:
For the existing implementation:

* Why is `std::mt19937_64` also used?

* Could `std::random_device` throw an exception?

* Should variables be `thread_local` and/or `static`?